isaric/parsers/isaric_washington.toml

```diff
@@ -0,0 +1,614 @@
+#:schema ../../schemas/dev/parser.schema.json
+
+[adtl]
+name = "washington"
+description = "COVID-19 patient data for ISARIC in SCCM VIRUS format, University of Washington Medical Center Northwest"
+
+[adtl.tables.subject]
+kind = "groupBy"
+groupBy = "subject_id"
+aggregation = "lastNotNull"
+schema = "../../schemas/dev/subject.schema.json"
+
+[adtl.tables.visit]
+kind = "groupBy"
+groupBy = "visit_id"
+aggregation = "lastNotNull"
+schema = "../../schemas/dev/visit.schema.json"
+
+[adtl.tables.observation]
+kind = "oneToMany"
+aggregation = "lastNotNull"
+schema = "../../schemas/dev/observation.schema.json"
+
+[adtl.defs]
+"Y/N/NK" = { values = { 0 = false, 1 = true } }
+admission_date = { field = "ADMIT DATE" }
+
+[study] 
+id = "CVMIRIJ"
+country_iso3 = "USA"
+
+[subject]
+country_iso3 = "USA"
+pathogen = "COVID-19"
+
+[subject.subject_id]
+field = "patient_study_id"
+sensitive = true
+
+[subject.enrolment_date] # optional field
+field = "date_filling_first_form"
+
+[subject.admission_date] # fill here!
+ref = "admission_date"
+
+[subject.age] # need fixing here for age '>90' to consider a character ..
+field = "age"
+# no age unit?
+
+[subject.sex_at_birth]
+field = "sex"
+values = { 1 = "Male", 2 = "Female", 3 = "Intersex", 4 = "Transgender" }
+
+[subject.ethnicity]
+field = "race"
+combinedType = "set"
+excludeWhen = "none"
+values = { 1 = "American Indian or Alaska Native", 2 = "Asian American", 3 = "Black or African American", 4 = "Native Hawaiian or Other Pacific Islander", 5 = "White Caucasian", 7 = "East Asian (China, Hongkong, Japan, S. Korea etc.)", 8 = "South Asian (India, Pakistan, Sri Lanka etc.)", 9 = "West Asian (including Arabic)", 10 = "South East Asian (Philippines, Thailand, Malaysia, Singapore, VietNam etc.)", 11 = "Mixed Race", 12 = "Unknown", 6 = "Other" }
+
+[subject.works_microbiology_lab]
+field = "employed_as_a_healthcare_w"
+description = "Patient Employed as a Healthcare Worker or in a Microbiology Laboratory?"
+ref = "Y/N/NK"
+
+[subject.works_healthcare]
+field = "employed_as_a_healthcare_w"
+description = "Patient Employed as a Healthcare Worker or in a Microbiology Laboratory?"
+ref = "Y/N/NK"
+
+[subject.has_tuberculosis]
+field = "any_other_pos___4"
+ref = "Y/N/NK"
+
+[subject.has_dementia]
+field = "comorbidities___34"
+ref = "Y/N/NK"
+
+[subject.has_obesity]
+field = "comorbidities___30"
+ref = "Y/N/NK"
+
+[subject.has_hiv]
+combinedType = "any"
+fields = [
+    { field = "comorbidities___22", ref = "Y/N/NK" },
+    { field = "any_other_pos___1", ref = "Y/N/NK" }
+]
+
+[subject.has_hypertension]
+field = "comorbidities___3"
+ref = "Y/N/NK"
+
+[subject.has_malignant_neoplasm]
+field = "comorbidities___20"
+ref = "Y/N/NK"
+
+[subject.has_smoking]
+field = "social_history"
+values = { 1 = "Current Smoler", 2 = "Former Smoker", 3 = "Vaping", 4 = "Alcohol use disorder", 5 = "Substance use disorder", 0 = "Unknown/Not available" }
```

Comment:
This needs to be a combined field, the headers are social_history___1, social_history___2 etc. Also in this case we're only looking at smoking - alcohol/substance/unkown should not be filled in. Not sure if we want to include vaping - we haven't up until now.

Reply:
I think I have fixed this ...